wallet-new/test/DevelopmentSpec.hs

```diff
@@ -0,0 +1,78 @@
+{-# OPTIONS_GHC -fno-warn-orphans       #-}
+
+{-# LANGUAGE ConstraintKinds            #-}
+{-# LANGUAGE FlexibleContexts           #-}
+{-# LANGUAGE FlexibleInstances          #-}
+{-# LANGUAGE MultiParamTypeClasses      #-}
+{-# LANGUAGE NoImplicitPrelude          #-}
+{-# LANGUAGE OverloadedStrings          #-}
+{-# LANGUAGE ScopedTypeVariables        #-}
+{-# LANGUAGE TypeFamilies               #-}
+{-# LANGUAGE TypeSynonymInstances       #-}
+{-# LANGUAGE UndecidableInstances       #-}
+
+-- Spec for testing `development` endpoints
+module DevelopmentSpec where
+
+import           Universum
+
+import           Data.Default (def)
+import           Pos.Client.KeyStorage (addSecretKey, getSecretKeysPlain)
+
+import           Pos.Util.BackupPhrase (BackupPhrase (..), safeKeysFromPhrase)
+import           Pos.Util.CompileInfo (HasCompileInfo, withCompileInfo)
+import           Pos.Launcher (HasConfigurations)
+import           Pos.Util.QuickCheck.Property (assertProperty)
+
+import           Test.Pos.Wallet.Web.Mode (walletPropertySpec)
+import           Test.Hspec (Spec, describe)
+import           Test.Hspec.QuickCheck (modifyMaxSuccess)
+import           Test.Pos.Configuration (withDefConfigurations)
+
+import           Cardano.Wallet.API.Development.LegacyHandlers (deleteSecretKeys)
+import           Cardano.Wallet.Server.CLI (RunMode (..))
+import           Servant
+
+{-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
```

Comment:
No need to write this -- CI won't fail on this as this is just a warning, AFAIK.

Reply:
You do need that, because the CI was failing yesterday.

Reply:
You're right, my bad, I thought it was a different warning. But now that you make me think about it, where is the code duplication stemming from? Any way we can make hlint happy by reducing it?

Reply:
`hlint` did care about following duplicated lines of code:
```
sKey <- either  (error "creating a secret key failed")
                         (pure . fst)
                         $ safeKeysFromPhrase mempty phrase
```

Here https://github.com/input-output-hk/cardano-sl/blob/938a4faca8a15e77ece01c03bdee82a89adb74b2/wallet-new/test/DevelopmentSpec.hs#L53-L55
and here https://github.com/input-output-hk/cardano-sl/blob/938a4faca8a15e77ece01c03bdee82a89adb74b2/wallet-new/test/DevelopmentSpec.hs#L67-L69

I would suggest to ignore that duplication and just using that annotation to make `hlint` happy.